drake/systems/plants/IKoptions.h

```diff
@@ -63,7 +63,7 @@ class drakeIKoptions_DLLEXPORT IKoptions
     void setqd0(const Eigen::VectorXd &lb, const Eigen::VectorXd &ub);
     void setqdf(const Eigen::VectorXd &lb, const Eigen::VectorXd &ub);
     void setAdditionaltSamples(const Eigen::RowVectorXd &t_samples);
-    void updateRobot(const RigidBodyTree * robot);
```

Comment:
the `const` version of this method wasn't implemented in the .cpp file, and the duplication in the header was causing weird swig issues